src/common_swagger_api/schema/containers.clj

```diff
@@ -0,0 +1,86 @@
+(ns common-swagger-api.schema.containers
+  (:use [common-swagger-api.schema :only [->optional-param describe]])
+  (:require [schema.core :as s]))
+
+(s/defschema Image
+  (-> {:name                            s/Str
+       :id                              s/Uuid
+       (s/optional-key :tag)            s/Str
+       (s/optional-key :url)            (s/maybe s/Str)
+       (s/optional-key :deprecated)     Boolean
+       (s/optional-key :auth)           (s/maybe s/Str)
+       (s/optional-key :osg_image_path) (s/maybe s/Str)}
+      (describe "A map describing a container image.")))
+
+(s/defschema Settings
+  (-> {(s/optional-key :cpu_shares)        (describe Integer "The shares of the CPU that the tool container will receive")
+       (s/optional-key :pids_limit)        Integer
+       (s/optional-key :memory_limit)      (describe Long "The amount of memory (in bytes) that the tool container is restricted to")
+       (s/optional-key :min_memory_limit)  (describe Long "The minimum about of memory (in bytes) that is required to run the tool container")
+       (s/optional-key :min_cpu_cores)     (describe Double "The minimum number of CPU cores needed to run the tool container")
+       (s/optional-key :max_cpu_cores)     Double
+       (s/optional-key :min_disk_space)    (describe Long "The minimum amount of disk space needed to run the tool container")
+       (s/optional-key :network_mode)      (describe s/Str "The network mode for the tool container")
+       (s/optional-key :working_directory) (describe s/Str "The working directory in the tool container")
+       (s/optional-key :name)              (describe s/Str "The name given to the tool container")
+       (s/optional-key :entrypoint)        (describe s/Str "The entrypoint for a tool container")
```

Comment:
These descriptions came from the corresponding individual schemas in `apps.routes.schemas.containers`.